assets/src/util/dateTime.ts

```diff
@@ -0,0 +1,34 @@
+export const dateFromEpochSeconds = (time: number): Date =>
+  new Date(time * 1_000)
+
+export const now = (): Date => new Date()
```

Comment:
I wonder if it would be possible for this file's interface to work entirely with seconds instead of working with a mix of `Date` and seconds. The fact that this file uses `Date` under the hood would then be an opaque implementation detail.

I think you could get by with only exporting 3 functions: `nowEpochSeconds()`, `formattedTime(epochSeconds: number)` and `formattedTimeDiff(epochSeconds1, epochSeconds2)`.

Reply:
I hesitate to use epoch seconds more since that isn't a "normal" unit in Javascript. This actually kind of makes me wonder: maybe we should be converting from epoch seconds to Date when we ingest the block waiver data? Then we could get rid of the redundant feeling `formatEpochSeconds` function and not have to worry about a different time standard anywhere else in JS land. How would you feel about that change?

Reply:
I made that change.